analyzer/codechecker_analyzer/buildlog/log_parser.py

```diff
@@ -418,16 +419,15 @@ def get_compiler_standard(compiler, language):
 
         standard = ""
         with tempfile.NamedTemporaryFile(
+                mode='w+',
                 suffix=('.c' if language == 'c' else '.cpp')) as source:
 
             with source.file as f:
                 f.write(VERSION_C if language == 'c' else VERSION_CPP)
 
             try:
-                proc = subprocess.Popen([compiler, source.name],
-                                        stdout=subprocess.PIPE,
-                                        stderr=subprocess.PIPE)
-                _, err = proc.communicate()  # Wait for execution.
+                err = ImplicitCompilerInfo.\
+                    __get_compiler_err(" ".join([compiler, source.name]))
```

Comment:
`__get_compiler_err` can return None if there is some OSError, will that cause problems with the `re.search` later?

Previously it did not run re.search if there was an exception.

Reply:
Any thoughts on this?

Reply:
Agreed.